packages/ui/src/routes/PopupPage.tsx

```diff
@@ -167,17 +167,32 @@ const PopupPage = () => {
             >
                 <div className="flex flex-row items-center justify-between w-full">
                     <div className="flex flex-row items-center space-x-3">
-                        <Link
-                            to="/accounts"
-                            className="transition duration-300"
-                            draggable={false}
-                            data-testid="navigate-account-link"
-                        >
-                            <AccountIcon
-                                className="w-8 h-8 transition-transform duration-200 ease-in transform hover:rotate-180"
-                                fill={getAccountColor(account?.address)}
-                            />
-                        </Link>
+                        <div className="relative flex flex-col items-start group">
+                            <Link
+                                to="/accounts"
+                                className="transition duration-300"
+                                draggable={false}
+                                data-testid="navigate-account-link"
+                            >
+                                <AccountIcon
+                                    className="w-8 h-8 transition-transform duration-200 ease-in transform hover:rotate-180"
+                                    fill={getAccountColor(account?.address)}
+                                />
+                            </Link>
+                            {/* Switch Account Icon ToolTip */}
+                            <div
```

Comment:
There is already a `Tooltip` component

Reply:
Used it now! Thanks for letting me know. 💪🏼